<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>设置对象</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rBtnSource">
           <property name="text">
            <string>listSource</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtnList">
           <property name="text">
            <string>listWidget</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtnTree">
           <property name="text">
            <string>treeWidget</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtnTable">
           <property name="text">
            <string>tableWidget</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>拖放参数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkBoxAcceptDrops">
           <property name="text">
            <string>acctptDrops</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="chkBoxDragEnabled">
           <property name="text">
            <string>dragEnabled</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>DrapDropMode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cBoxDropMode">
           <item>
            <property name="text">
             <string>NoDragDrop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DragOnly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DropOnly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DragDrop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InternalMove</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>DrapDropAction</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cBoxDropAction">
           <item>
            <property name="text">
             <string>CopyAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MoveAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LinkAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IgnoreAction</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>257</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxSource">
       <property name="title">
        <string>listSource</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidgetSource">
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/delete (3).png</normaloff>:/img/delete (3).png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/diskette.png</normaloff>:/img/diskette.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/folder (5).png</normaloff>:/img/folder (5).png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/smart-tv.png</normaloff>:/img/smart-tv.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新建项目</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/img/gallery.png</normaloff>:/img/gallery.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxList">
       <property name="title">
        <string>listWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxTree">
       <property name="title">
        <string>treeWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxTable">
       <property name="title">
        <string>tableWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
